idunn/api/urls.py

```diff
@@ -74,6 +75,11 @@ def get_api_urls(settings):
             response_model=IdunnAutocomplete,
             response_model_exclude_unset=True,
         ),
+        APIRoute(
+            "/search",
+            search,
+            response_model=IdunnAutocomplete,
+        ),
```

Comment:
For consistency with "/autocomplete", `methods` and `response_model_exclude_unset` should be defined too.